share/functions/fish_vi_key_bindings.fish

```diff
@@ -1,6 +1,42 @@
 alias fish_vi_dec 'fish_vi_inc_dec dec'
 alias fish_vi_inc 'fish_vi_inc_dec inc'
 
+function fish_vi_arg_digit --description 'Accumulate a digit for the next command'
+    set -g __fish_vi_count "$__fish_vi_count$argv[1]"
+end
+
+function fish_vi_run_count --description 'Run a command $__fish_vi_count times'
+    set -l count (string join '' $__fish_vi_count)
```

Comment:
I'm not sure why the string-join would be necessary, this should work:

	set -l count $__fish_vi_count

I guess you maybe added it for input sanitization.
If we want to sanitize it, we should go all the way (check if the arg is a single number).
I'd say it's not necessary.